### DISCUSSION TITLE
js chunck

### DISCUSSION THREAD

**Discussion by wilcar** [quarto-dev/quarto-cli#2901] [Q&A]
I want to add a javacript function code chunk in website quarto with RStudio and R 

```{js}
$(function(){
    $(window).scroll(function(){
        if($(this).scrollTop() > 100){
            $('header').addClass('fixed-header');
        }
        else{
            $('header').removeClass('fixed-header');
        }
    });
});
```

but I get an error with the following message: 

        Starting python3 kernel...Traceback (most recent call last):
          File "C:\Users\----\AppData\Local\Programs\Quarto\share\jupyter\jupyter.py", line 21, in <module>
            from notebook import notebook_execute, RestartKernel
          File "C:\Users\cariou-w\AppData\Local\Programs\Quarto\share\jupyter\notebook.py", line 16, in <module>
            import nbformat
        ModuleNotFoundError: No module named 'nbformat'
        Python 3 installation:
          Version: 3.9.5
          Path: C:/Users/-----/AppData/Local/Programs/Python/Python39/python.exe
          Jupyter: (None)
        
        Jupyter is not available in this Python installation.
        Install with py -m pip install jupyter

Translated with www.DeepL.com/Translator (free version)

**Comment by cscheid** [comment]
You probably attempted to use this syntax:

````
```{javascript} (or {js}?)
...
```
````

That is syntax for *render-time* execution. If you want javascript code to run on the *webpage*, simply add a `<script>` tag to the markdown.
